.gitignore

```diff
@@ -12,3 +12,11 @@ Kwarchart-Example/Kwarchart-Example.xcodeproj/project.xcworkspace/xcshareddata/I
 kwarchart.xcodeproj/project.xcworkspace/xcshareddata/WorkspaceSettings.xcsettings
 kwarchart.xcodeproj/project.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist
 kwarchart.xcodeproj/xcshareddata/xcschemes/kwarchart-Package.xcscheme
+
+## Github store
+.DS_Store
+
+## Fastlane
+fastlane/report.xml
+fastlane/test_output/report.junit
+fastlane/test_output/report.html
```

Comment:
I think it is better to add the [Screenshot directory](https://github.com/jaimejazarenoiii/kwarchart-ios/blob/5057a2b80af2f7d6585536cab75649188f41d891/fastlane/Snapfile#L21), since images are heavy

Reply:
I'm sorry but I'm going to oppose this, because the sole purpose for that is to have a presentation of the implemented UI, without even running the app, so that QA and designer also can easily check view. Also reviewer can easily check the UI Changes from checking diff